examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -189,6 +189,9 @@ service ABitOfEverythingService {
 		}
 	};
 
+  // Create a new ABitOfEverything
+  //
+  // This API creates a new ABitOfEverything
```

Comment:
Oops, I left that comment on the commit directly -- sorry. Can you make the indentation match the rest of the file?

Reply:
Oh, sorry for the mistakes. I fixed it by https://github.com/grpc-ecosystem/grpc-gateway/pull/667/commits/4b6f8895524282af806fda06e42705cbe400b817